doc.go

```diff
@@ -198,4 +198,8 @@
 // match AnyInt struct created by NewAnyInt() method. AnyInt struct will match
 // any integer passed to redigomock from the tested method. Please see
 // fuzzyMatch.go file for more details.
+//
+// The method Conn.Do is safe for concurrent use, but other methods and fields,
+// such as Command and Errors, are generally not and must only be accessed with
+// external synchronization.
```

Comment:
The `Do` command also read/write other attributes like `commands` and `replies` that don't have locks for concurrent use. 🤔 If we are adding this comment we probably should also take care of them.